src/zino/tasks/juniperalarmtask.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from typing import Literal
+
+from zino.snmp import SNMP
+from zino.statemodels import AlarmEvent, EventState
+from zino.tasks.task import Task
+
+_logger = logging.getLogger(__name__)
+
+
+class JuniperAlarmTask(Task):
+    """Checks yellow and red alarm count for juniper device."""
+
+    async def run(self):
+        device_state = self.state.devices.get(self.device.name)
+        if not device_state.is_juniper:
+            return
+
+        try:
+            yellow_alarm_count, red_alarm_count = await self._get_juniper_alarms()
+        except TypeError:
+            return
+
+        device_state.alarms = {
+            "yellow": yellow_alarm_count,
+            "red": red_alarm_count,
+        }
+
+        self.create_alarm_event(color="yellow", alarm_count=yellow_alarm_count)
+        self.create_alarm_event(color="red", alarm_count=red_alarm_count)
+
+    async def _get_juniper_alarms(self):
+        snmp = SNMP(self.device)
+        yellow_alarm_count = await snmp.get("JUNIPER-ALARM-MIB", "jnxYellowAlarmCount", 0)
+        red_alarm_count = await snmp.get("JUNIPER-ALARM-MIB", "jnxRedAlarmCount", 0)
+        if yellow_alarm_count:
+            yellow_alarm_count = yellow_alarm_count.value
+        if red_alarm_count:
+            red_alarm_count = red_alarm_count.value
+
+        if type(yellow_alarm_count) is not int or type(red_alarm_count) is not int:
+            _logger.error(
+                "Device %s returns alarm count not of type int, type yellow alarm: %s, type red alarm count: %s",
```

Comment:
final comment i think: for yellow it says `yellow alarm` and for red it says `red alarm count`